### DISCUSSION TITLE
Using AltGr for hotkeys

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#383] [Help] [answered]
I'm confused about AltGr. 

When I set it as a hotkey in FP (for example AltGr+Space), it recognizes as Ctrl+Alt (for example Ctrl+Alt+Space). I understand that Ctrl+Alt=AltGr, right? When I input the hotkey in the map using Ctrl+Alt, it works fine, but when I input the hotkey using AltGr, nothing happens. 

Is it a bug, or a limitation of the AltGr?

**Comment by macmarrum** [comment]
1. Are you on Windows?
2. What keyboard layout do you use?
3. How do you use AltGr for regular typing? 
 a. When typing some text, does Ctrl+Alt+letter have the same effect as AltGr+letter (e.g. adding an accent or similar)?

**Comment by euu2021** [comment]
Searching about it, I concluded that i't a bug, so I will open a bug report with the details. Thanks,

**Comment by Illegit** [comment]
see issue #290.
illegit

**Comment by dpolivaev** [comment, marked as answer]
https://github.com/freeplane/freeplane/issues/384
